app/api_clients/github.js

```diff
@@ -29,6 +32,31 @@ class GithubAPIClient extends GithubAPI {
 
       return Promise.resolve(user.github_access_token);
     };
+
+    async function paginate(method, params = {}) {
```

Comment:
I couldn't just assign an anonymous `async` function to `this`, that's why it's named